addons/l10n_in/data/res_country_state_data.xml

```diff
@@ -8,6 +8,13 @@
         <field name="l10n_in_tin">97</field>
     </record>
 
+    <record id="state_in_oc" model="res.country.state">
+        <field name="name">Other Country</field>
+        <field name="code">IN_OC</field>
+        <field name="country_id" ref="base.in"/>
+        <field name="l10n_in_tin">96</field>
+    </record>
```

Comment:
What is the difference between Other Country and Other Territory?  It does not make sense to have that as a state in India.  False could mean one of them...

Reply:
both are different so we can't assume that False could mean one of them.
Also, we need to print that value on the invoice and it's mandatory.
You can check this for legal requirements https://www.cbic.gov.in/resources//htdocs-cbec/gst/tax%20Invoice%20and%20other.pdf 

you can also check State Codes here https://einvoice1.gst.gov.in/Others/MasterCodes

Reply:
@jva-odoo But if the country is different than India and it is False, it needs to be this, right?  And in the print-out on the report, you could put a similar logic...

Reply:
@jco-odoo There is many complex rules for this so we need to add if.
exp: Chapter V -> sub-section 10(d) says
"where the goods are assembled or installed at site, the place of supply shall
be the place of such installation or assembly;"

So if I will to customer in Indian but it's install in outside the country then place of supply is "Other Country"
And same for Services where  

see rules here https://www.cbic.gov.in/resources//htdocs-cbec/gst/igst-act.pdf;jsessionid=63D695F612B9972934E8131415FEBA28

Reply:
@jco-odoo in the above case GST treatment is registered because I invoiced to an Indian registered business.
So can't get GST treatment.
So we can add a new model if you think we can add it in states.
because it's mandatory for accounting in Indian.